src/main/java/com/zpi/token/domain/authorizationRequest/request/RequestErrorType.java

```diff
@@ -2,37 +2,44 @@
 
 public enum RequestErrorType {
     UNAUTHORIZED_CLIENT {
```

Comment:
remove comments, you can add variable and constructor to enum and then UNAUTHORIZED_CLIENT("UNAUTHORIZED_CLIENT)"

Reply:
- [ ] fix